src/webhook.rs

```diff
@@ -732,11 +688,151 @@ async fn handle_comment(
 				number,
 			))))?;
 		}
+	} else if body.to_lowercase().trim() == BURNIN_REQUEST.to_lowercase().trim()
+	{
+		// Check the user is a member of the org
+		let member = github_bot
+			.org_member(&owner, &requested_by)
+			.await
+			.map_err(|e| {
+				Error::OrganizationMembership {
+					source: Box::new(e),
+				}
+				.map_issue(Some((
+					owner.to_string(),
+					repo_name.to_string(),
+					number,
+				)))
+			})?;
+		if member != 204 {
```

Comment:
`204` seems to be a magic number. How about putting it into a constant thus making it self-explanatory via the constant name?

Reply:
When I saw this block of code checking for org membership, I wanted to pull it out into a separate function. But it seemed like `github_bot.org_member()` already is that function so I assumed there is a reason for this check being somewhat verbose and didn't want to mess with it without understanding that and risk introducing a security hole. Anyway, just copying the block was lazy. I'll try to improve it and at least make this a constant.

Reply:
So the reason why this block was copied all over `handle_comment` probably was because the simple act of extracting a chunk of code into a function can turn into a deep rabbit hole for Rust beginners...

IMHO [my first approach](https://gist.github.com/haikoschol/41c1caa55c252c4b42909a7600ead494) was not terrible, but unfortunately [the compiler is not ready for it yet](https://github.com/rust-lang/rust/issues/62290). So instead I opted for a somewhat ugly struct.